src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,71 @@
     }
   }
 
+  // Link and Action Buttons
+  &.pf-m-link,
+  &.pf-m-action {
+    background-color: transparent;
+    box-shadow: none;
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    &.pf-m-focus {
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
+
+  // Link Buttons
+  &.pf-m-link {
+    display: inline-flex;
```

Comment:
why are we changing the display of a btn?

Reply:
Looking of this specific situation and assuming that in many cases an icon will be present, with inline-flex the content can align vertically.

<img width="321" alt="screen shot 2018-06-05 at 12 53 59 pm" src="https://user-images.githubusercontent.com/5385435/40990684-a0897a12-68bf-11e8-9205-95c28123f384.png">

Reply:
@andresgalante @christianvogt you're right, this is superfluous